packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -143,14 +143,6 @@ abstract class BindingBase {
         name: 'exit',
         callback: _exitApplication,
       );
-      registerServiceExtension(
```

Comment:
Is this just no longer a thing - or why remove it?

Reply:
its not a thing